bokehjs/src/coffee/core/bokeh_events.ts

```diff
@@ -0,0 +1,200 @@
+import { logger } from "./logging"
+
+export class Event {
+    static _event_classes = {}
+
+    constructor(options = {}) {
+        this._options = options;
+        if (options.model_id) {
+            this.model_id = null
+        }
+    }
+
+    set_model_id(id) {
```

Comment:
And if unsure or there is no good type available, you can just use `any`. There are a lot of errors regarding implicit `any`, so if you make this explicit, they will go away. We don't want to have any `any` types floating around, but at this point we can allow for them.